### PR TITLE
Update publisher metadata from 'Kiwix' to 'openZIM'

### LINKED ISSUE #291: Harmonize default publisher to `openZIM`
As required in https://github.com/openzim/zim-requests/issues/779, the default ZIM publisher must be `openZIM` (mind the case)

### PR DESCRIPTION
**Pull request by dan-niles**
Resolves #291
Updated publisher metadata to 'openZIM' in `entrypoint.py` and `scraper.py`.

### CODE REVIEW

File: src/sotoki/entrypoint.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ def main():
         "-p",
         "--publisher",
         help="Custom publisher name (ZIM metadata). “OpenZIM” otherwise",
-        default="Kiwix",
+        default="openZIM",
```
Comment (benoit74) [inline src/sotoki/entrypoint.py:116]: I know it was here before you came, but please remove this default value, we should not have two default values in two different locations, one is enough

**Review by benoit74 — CHANGES_REQUESTED** [review]
Please fix comment and add an entry to CHANGELOG.md, in the "Unreleased" section ("Fixed" or "Changed" subsection, not sure which one is more appropriate, I let you choose, it is not that important anyway)

File: src/sotoki/entrypoint.py (hunk 1 above)
Comment (dan-niles) [reply src/sotoki/entrypoint.py]: @benoit74 I have made the changes you requested. Can you please review it?

Comment (benoit74) [reply src/sotoki/entrypoint.py]: Thank you for the ping, looks like the notification slipped through my attention. I will have a look at it right now.

**Review by benoit74 — APPROVED** [review]
LGTM

**Pull request merged**